src/gafaelfawr/cli.py

```diff
@@ -84,3 +76,7 @@ def init(settings: str) -> None:
     config_dependency.set_settings_path(settings)
     config = config_dependency()
     initialize_database(config)
+
+
+if __name__ == "__main__":
```

Comment:
Is this a hold-over from the previous iteration, or is there something that still needs to directly execute this module?

Reply:
Yeah, it's directly executed in `prestart.sh` to initialize the database (although that will eventually be replaced with Alembic) and it's directly executed in `tox -e run` to start a development server. It's also there to support `gafaelfawr generate-key`.

Reply:
I thought the `gafaelfawr` command-line tool was created by the entry_point in setup.cfg? https://github.com/lsst-sqre/gafaelfawr/blob/d68134dce5c5eb1036232b201b7be0f767633892/setup.cfg#L37-L39

Reply:
Oh, right, yes, this was only because of the workaround I was using for `prestart.sh` that isn't needed. Thanks, good catch, I forgot about that. Fixed in https://github.com/lsst-sqre/gafaelfawr/pull/131.